stacks-signer/src/signer.rs

```diff
@@ -409,6 +409,39 @@ impl Signer {
         self.coordinator_selector.last_message_time = Some(Instant::now());
     }
 
+    /// Resend outbound messages if no state change has occurred within the configured timeout
+    pub fn resend_outbound_messages(&mut self, response_wait_timeout: Duration) {
+        // We do not need to resend outbound messages if we are in the Idle state.
```

Comment:
Would it be the case then that `self.signer_db.outbound_messages_lookup(self.reward_cycle)` would return `None` if the state were `Idle`?